server/backends/github/github.go

```diff
@@ -256,3 +250,8 @@ func getCookie(r *http.Request, name string) string {
 	}
 	return ""
 }
+
+func redirectWithError(w http.ResponseWriter, r *http.Request, err error) {
+	log.Warningf(err.Error())
```

Comment:
uberubernit: here and other places, you don't need the "*f" variants of status errors or log functions if you're not doing formatting. so this could be log.Warning(err.Error())

this is such a nit i feel bad after even writing this comment.

Reply:
lol fixed, my logic is always just been to use the `f` version everywhere because it's an easy change when you inevitably want to print out some extra random crap